tests/system/test_requests.py

```diff
@@ -109,10 +109,6 @@ def test_ok(self):
         r = self.request_intake()
         assert r.status_code == 202, r.status_code
 
-    def test_sourcemap_upload(self):
-        r = self.upload_sourcemap(file_name='bundle.js.map')
-        assert r.status_code == 202, r.status_code
```

Comment:
Why is this removed, is it covered somewhere else?

Reply:
yeah, in many places, eg https://github.com/elastic/apm-server/blob/502f15587e048ef508f32e41a3b78cb2ea8cfe10/tests/system/test_integration.py#L332